### ISSUE TITLE
How to deal with "missing href in anchor" linter errors when using tabs or dropdowns

### ISSUE THREAD

**Issue by svdHero** [jgthms/bulma#3548]
This is about **Bulma**.
I need help.

### Overview of the problem

This is about the Bulma **CSS framework**
This is about the Bulma **Docs**
I'm using Bulma **version** [0.9.4]
My **browser** is: Chrome Version 103.0.5060.53 (Official Build) (64-bit)

This is a repost of #3325 

### Description

Whenever I use components like, e.g., [tabs](https://bulma.io/documentation/components/tabs/) or [dropdowns](https://bulma.io/documentation/components/dropdown/) where the entries are anchor elements `<a>`, I get a linter error

> The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

I tried to use button elements of class `is-ghost` instead of the anchor element, but at least for the tabs, these do not get rendered properly.

What would be the best practice to add tabs and dropdown entries that "feel" like hyperlinks (mouse cursor, etc.) and make the linter happy? Did I miss anything in the docs?

**Comment by jgthms** [comment]
I agree that buttons should be available here. This is something to work on.

**Comment by Mark-777-0** [comment]
I can work on this @jgthms , would you assign it to me?

**Comment by MurdoMaclachlan** [comment]
Just want to drop in on here and say I've also run into this recently.

It's a pretty big accessibility issue, since if, like with mine, the website design heavily uses tabs, much of the website ends up completely unavailable to keyboard-only users.

In the mean-time, I've found adding `href="#" onclick="return false;"` to the `<a>` elements works as a hack, providing keyboard accessibility while preventing the page from linking anywhere or refreshing, but I think a solution allowing use of `<button>` for these components would be much cleaner.

**Comment by dseebacher** [comment]
If there might be viable tags besides `a` and `button`, I would suggest a class based approach instead, similar to the button styling.
